crates/prql-compiler/tests/integration/queries/read_csv.prql

```diff
@@ -0,0 +1,5 @@
+# sqlite:skip
+# postgres:skip
+# mysql:skip
+# clickhouse:skip (Not implemented yet)
+from (read_csv "data_file_root/media_types.csv")
```

Comment:
I think it's really good that we're testing this — one of the most helpful things our integration tests can do!